odoo/addons/base/tests/config/14.0.conf

```diff
@@ -32,6 +32,7 @@ log_db_level = warning
 log_handler = :INFO
 log_level = info
 logfile = 
+logfilesdb = 
```

Comment:
Read the comment on top of this file. Please do not modify it. We use it to make sure an old configuration file still work with modern Odoo.